comskip.c

```diff
@@ -14340,7 +14363,7 @@ void ProcessCSV(FILE *in_file)
             */
         }
 
-        if ((frame[i].schange_percent < 20) && (black[black_count - 1].frame != i))
+        if ((frame[i].schange_percent < 20) && (black[black_count == 0 ? 0 : black_count - 1].frame != i))

```

Comment:
Line 14072 has a printf to stderr (I assume debugging output). 
Should this be done?